tests/topotests/bfd_vrflite_topo1/r1/zebra.conf

```diff
@@ -0,0 +1,24 @@
+vrf vrf1
+ ip route 2.2.2.2/32 192.168.0.2
```

Comment:
And Orange too :smile:

Reply:
1.1.1.1 and 2.2.2.2  are used a lot on topotests. Did I miss something in the remark ?

Reply:
docs.frrouting.org/projects/dev-guide/en/latest/topotests.html?#writing-tests:

^f (RFC 5737)

Reply:
Old tests weren't changed...

Reply:
topotest is not exactly documentation.
this said, I expect that private networks are not impacted.
let s try with this.